examples/server/server.cpp

```diff
@@ -1736,7 +1812,7 @@ struct llama_server_context
 
                     if (has_images && !ingest_images(slot, n_batch))
                     {
-                        LOG_TEE("failed processing images\n");
+                        LOG_WARNING("failed processing images", {});
```

Comment:
Here it should be `LOG_ERROR`, since we drop the task.

`LOS_WARNING` should be more suitable for situation like "we will still do the task, but not optimal"

Reply:
You are right there is a code smell here. Early eturning within `update_slots` make the server in an unstable state. Need to be fixed later on with an appropriate tests.